apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx

```diff
@@ -48,7 +49,12 @@ export function TransferNFTForm({ objectId }: { objectId: string }) {
         },
         onSuccess: (response) => {
             queryClient.invalidateQueries(['object', objectId]);
-            queryClient.invalidateQueries(['objects-owned']);
+            queryClient.invalidateQueries([
```

Comment:
while this seems like the correct fix, I think manually plugging in the query keys used by `getOwnedObjects` here is super error-prone. Can we perhaps make a query key factory like https://tkdodo.eu/blog/effective-react-query-keys#use-query-key-factories instead?

Reply:
You are right, I will add this to wallet hardening epic